index.html

```diff
@@ -1753,6 +1753,105 @@ <h3>
           </pre>
         </section>
 
+          <section class="normative">
+              <h3>
+                  Batch Operations
+              </h3>
+
+              <p>
+                  Documents within a vault can be created, updated or deleted in a single REST call.
+                  The following HTTP status codes are defined for this service:
+              </p>
+
+              <table class="simple">
+                  <thead>
+                  <th style="white-space: nowrap">HTTP Status</th>
+                  <th>Description</th>
+                  </thead>
+                  <tbody>
+                  <tr>
+                      <td>200</td>
+                      <td>
+                          All operations successful.
+                      </td>
+                  </tr>
+                  <tr>
+                      <td>400</td>
+                      <td>
+                          One or more operations were unsuccessful.
+                      </td>
+                  </tr>
+                  </tbody>
+              </table>
+
+              <p>
+                  A protocol example of a batch request is shown below. The first element in the JSON
+                  array represents a create document operation. The second element represents an update document operation.
+                  The third element represents a delete document operation.
+              </p>
+
+              <pre class="example highlight" title="Data vault batch operation request">
+  POST  /encrypted-data-vaults/z4sRgBJJLnYy HTTP/1.1
```

Comment:
Is the idea here not to have a separate batch API endpoint?

Reply:
Indeed... perhaps it should be `/encrypted-data-vaults/z4sRgBJJLnYy/batch`?

Reply:
+1 to separate endpoint. (Would result in simpler logic, on the server side, so it doesn't have to guess whether it's a batch request or a single request coming in)

Reply:
Done! (although - I believe that `/encrypted-data-vaults/z4sRgBJJLnYy` is actually not a used endpoint currently.... but I still think it's more consistent to have `batch` in the endpoint, similar to the queries one)